test/parallel/test-repl-definecommand.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+require('../common');
+
+const stream = require('stream'),
+      assert = require('assert'),
+      repl = require('repl');
+
+var output = '';
+const inputStream = new stream.PassThrough();
+const outputStream = new stream.PassThrough();
+outputStream.on('data', function(d) {
+  output += d;
+});
+
+const r = repl.start({
+  input: inputStream,
+  output: outputStream,
+  terminal: true
+});
+
+r.defineCommand('say1', {
+  help: 'help for say1',
+  action: function(thing) {
+    output = '';
+    this.write('hello ' + thing);
+    this.displayPrompt();
+  }
+});
+
+r.defineCommand('say2', function() {
+  output = '';
+  this.write('hello from say2');
+  this.displayPrompt();
+});
+
+inputStream.write('.help\n');
+assert(/\nsay1\thelp for say1\n/.test(output), 'help for say1 present');
```

Comment:
The second parameter will be thrown as an error, if the first param is not truthy. So the text should be negative I believe.